docs/user-guide/install-k8s.md

```diff
@@ -35,13 +35,12 @@ sections).
 Having said that, at the moment the Sysbox installation process described in
 this document has only been tested in the following scenarios:
 
-| Kubernetes Distros    |  Tested OS Distros                         | Cluster Setup Notes |
-| --------------------- | ------------------------------------------ | ------------------- |
-| Kubernetes (regular)  |  Ubuntu Bionic or Focal                    | [(1)](install-k8s-distros.md#kubernetes-regular) |
-| Amazon EKS            |  Ubuntu Focal                              | [(2)](install-k8s-distros.md#aws-elastic-kubernetes-service-eks) |
-| Google GKE            |  Ubuntu-Containerd or Ubuntu-Docker images | [(3)](install-k8s-distros.md#google-kubernetes-engine-gke) |
-| Rancher RKE           |  Ubuntu Focal                              | [(4)](install-k8s-distros.md#rancher-kubernetes-engine-rke) |
-
+| Kubernetes Distros    |  Tested OS Distros                          | Cluster Setup Notes |
+| --------------------- | ------------------------------------------- | ------------------- |
+| Kubernetes (regular)  |  Ubuntu Bionic / Focal, Flatcar             | [(1)](install-k8s-distros.md#kubernetes-regular) |
+| Amazon EKS            |  Ubuntu Focal, Flatcar                      | [(2)](install-k8s-distros.md#aws-elastic-kubernetes-service-eks) |
```

Comment:
Does EKS support Flatcar? I know we formed a K8s cluster with Flatcar using EC2 instances, but that's not the same as EKS.

Same for GKE below.

Reply:
Oops, must be me in sleep-deprived mode, only AKS supports Flatcar, will fix it.